src/auth.js

```diff
@@ -27,6 +27,7 @@ const config = {
       : 'https://sso-dev.pathfinder.gov.bc.ca',
   realmName: 'devhub',
   clientId: 'signing-web',
+  kcIDPHint: 'idir',
```

Comment:
Better to write a line of comment: The specifies to skip and use IDIR login page directly...

Reply:
Added.